### ISSUE TITLE
etcdctl snapshot save command is not working correctly

### ISSUE THREAD

**Issue by NiveditaKumari14** [etcd-io/etcd#12222]
When i am executing "etcdctl snapshot save" command getting error of:

sh-4.2# etcdctl snapshot save /tmp/ssnapshot.db
Error: snapshot must be requested to one selected node, not multiple [https://192.168.75.47:2379 https://192.168.75.15:2379 https://192.168.75.28:2379 https://192.168.75.52:2379]

After providing "--endpoints=[192.168.75.15:2379]", still getting error.

sh-4.2# etcdctl snapshot save /tmp/snshot.db --endpoints=[192.168.75.15:2379]
2020-08-14 12:50:24.624771 C | pkg/flags: conflicting environment variable "ETCDCTL_ENDPOINTS" is shadowed by corresponding command-line flag (either unset environment variable or disable flag)

 
Etcdctl version:
Version: 3.4.7
API Version: 3.4

**Comment by agargi** [comment]
@NiveditaKumari14 Can you please print the value for environment variable: ETCDCTL_ENDPOINTS? Looks like there is a conflict since you are both setting this up environment and providing the same input through a command-line option.

Try unsetting the environment variable and it should work.

**Comment by dbavatar** [comment]
I just ran into this upgrading from 3.3->3.4 which broke my cluster - now my etcdctl liveness check is failing with the same error. Setting both is exactly what I intended to do - I don't understand why this is an error - it should be at most a warning, but I'd say the cleanest thing to assume a command line argument is an override, if I run the command in verbose mode only then maybe it can show a warning. This is the way we (I think most people) expect Linux to behave, which I think @NiveditaKumari14 also expected. So I would like to continue to have both environment and add command line args as necessary which override them.

I'd be happy to submit a PR but this is really a philosophical issue, not a code issue. For example the same sort of thing is blocking me from upgrading upgrading to 3.4.12 - the data path permissions checking just decides to error out - it is not relevant for my configuration, but it means when I try to upgrade it breaks. See https://github.com/etcd-io/etcd/pull/12242

Broke in 748ab8c3989886c2145eea06703d2981a224024f @gyuho
